app.js

```diff
@@ -37,3 +38,147 @@ routes.init(app);
 app.use(errors.handle);
 
 module.exports = app;
+
+// ---------
+// ---------
+// ---------
+// ---------
+// ---------
+// ---------
+// ---------
+// ---------
+// ---------
+const request = require('request');
+
+// This URL should return JSON data
+const url = 'https://jsonplaceholder.typicode.com';
```

Comment:
Move this to /config/environment.js

And take it from ENV variables.

```
// /config/environment.js
const { ALMBUS_API_URL } = process.env || {};

module.exports = {ALMBUS_API_URL};
```

```
// /services/albums.js
const { ALMBUS_API_URL: url } = require('../config/environment') ;

```

This way that URL is going to be set as an ENV var.